src/main/java/org/example/Cocktail.java

```diff
@@ -0,0 +1,72 @@
+package org.example;
+
+public class Cocktail {
+    private final String base;
+    private final String mixer;
+    private final String sweetener;
+    private final String beauty;
+    private final int iceCount;
+
+    private Cocktail(final CocktailBuilder cocktailBuilder) {
+        this.base = cocktailBuilder.base;
+        this.mixer = cocktailBuilder.mixer;
+        this.sweetener = cocktailBuilder.sweetener;
+        this.beauty = cocktailBuilder.beauty;
+        this.iceCount = cocktailBuilder.iceCount;
+    }
+
+    @Override
+    public String toString() {
+        return String.format("Cocktail consists of: base - %s, mixer - %s, sweetener - %s, beauty - %s, with ice - %s", base, mixer, sweetener, beauty, iceCount);
+    }
+
+    public static class CocktailBuilder {
```

Comment:
From my view `Cocktail` is redundant. Just builder. In this case call will look `Cocktail.Builder`.
```suggestion
    public static class Builder {
```

Reply:
Done